scripts/microbe_stage/organelle.as

```diff
@@ -839,6 +839,11 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
         //sceneNode.entity.setColour(colour)
     }
 
+    void hideEntity(){
+        auto renderNode = world.GetComponent_RenderNode(organelleEntity);
+        if(renderNode !is null && renderNode.Node !is null)
+        renderNode.Node.removeFromParent();
```

Comment:
The formatting script doesn't properly work with multiple tabs. So there is one level of indentation missing here.